int32 roomID
string name
float32 battery_level
